setup_linux.Rmd

```diff
@@ -0,0 +1,73 @@
+---
+title: "Linux Setup"
+author:
+  name: "Thomas Roppelt"
+  affiliation: "Friedrich-Alexander-Universität Erlangen-Nürnberg (FAU)"
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+bibliography: library.bib
+---
+
+```{r setup, echo = FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+## Linux distributions (distros)
+Looking at Linux distributions and deciding for one can be a daunting task for newcomers.
+Here are some key differences between distributions:
+
+### Release Model
+The release model decides how frequently updates reach your system. Frequent smaller updates are called rolling release. If only very big updates are released, it is a point or major release model.
```

Comment:
another small todo: it'd be nice to have *every sentence one line*. Makes for nicer git diffs, going forward.

Reply:
will fix :)